Formula/kontena-cli.rb

```diff
@@ -0,0 +1,68 @@
+class KontenaCli < Formula
+  desc "Command-line client for Kontena container & microservices platform"
+  homepage "https://kontena.io/"
+  url "https://github.com/kontena/kontena.git", :tag => "v1.3.4"
+  head "https://github.com/kontena/kontena.git"
+
+  bottle :unneeded
+
+  depends_on :ruby => "2.1"
+
+  def install
+    # Build the gem from sources and install it
+    cd "cli" do
+      system "gem", "build", "--norc", "kontena-cli.gemspec"
+      system "gem",
+        "install", Dir["kontena-cli-*.gem"].first,
+        "--install-dir", buildpath/"out",
+        "--wrappers",
+        "--env-shebang",
+        "--no-document",
+        "--norc"
+    end
+
+    (bin/"kontena").write(exec_script)
+
+    rm_rf buildpath/"out/cache"
+
+    libexec.install Dir["out/*"]
+
+    # Write a .ruby-version file to match the current ruby version (for rbenv/rvm users)
+    system "ruby", "-e", "File.write('" + libexec/"bin/.ruby-version" + "', RUBY_VERSION)"
+
+    zsh_completion.install "cli/lib/kontena/scripts/kontena.zsh" => "_kontena"
+    bash_completion.install "cli/lib/kontena/scripts/kontena.bash" => "kontena"
+  end
+
+  def exec_script
+    <<-EOS.undent
+      #!/bin/bash
+
+      export GEM_PATH=#{libexec}
+      export KONTENA_EXTRA_BUILDTAGS=homebrew#{",head" if build.head?}
+      exec #{libexec/"bin/kontena"} "$@"
+    EOS
+  end
+
+  def caveats
+    unless Dir["/usr/local/{opt,var}/rbenv/shims/kontena"].empty?
+      <<-EOS.undent
+        You seem to use rbenv and have a previously installed kontena-cli that
+        may get loaded instead of the homebrew installed version.
+
+        To uninstall the previous installation copy and paste this into the
+        terminal:
+
+          for ruby in $(rbenv whence kontena); do \\
+            rbenv shell $ruby; gem uninstall --force -a -x kontena-cli; \\
+          done; \\
+          rbenv rehash
+      EOS
+    end
+  end
+
+  test do
+    assert_match "+homebrew", shell_output("#{bin}/kontena --version")
```

Comment:
needs a test that exercise and verifies the functionality of the software

Reply:
Added some, but most of the app functionality requires access to a running server installation.

Reply:
You can probably mock that. See the `faas-cli` formula's test block.